tests/test.bats

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env batsPI
+
+load test_helper
+
+HOST="ckan"
```

Comment:
```suggestion
HOST="ckan-dev"
```
If we are going to use _ckan-dev_ for testing we will need to change this. 
Definition required

Reply:
how about "catalog-next"? :-)

Reply:
@avdata99 if I use this I get 
```
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'                                                                                                                                                  
test_1        | # nc: bad address 'ckan-dev'
test_1        | # nc: bad address 'ckan-dev'
test_1        | # nc: bad address 'ckan-dev'
test_1        | # nc: bad address 'ckan-dev'
test_1        | # nc: bad address 'ckan-dev'
```